s/sync/sync_ubi_8.5.sh

```diff
@@ -0,0 +1,134 @@
+#!/bin/bash -e
+# -----------------------------------------------------------------------------
+#
+# Package       : sync
+# Version       : master(036812b2)
+# Source repo   : https://github.com/golang/sync.git
+# Tested on     : ubi 8.5
+# Language      : go
+# Travis-Check  : true
+# Script License: Apache License, Version 2 or later
+# Maintainer    : Sachin K {sachin.kakatkar@ibm.com}
+#
+# Disclaimer: This script has been tested in root mode on given
+# ==========  platform using the mentioned version of the package.
+#             It may not work as expected with newer versions of the
+#             package and/or distribution. In such case, please
+#             contact "Maintainer" of this script.
+#
+# ----------------------------------------------------------------------------
+#Run the script:./sync_ubi_8.5.sh master(version_to_test)
+PACKAGE_NAME=sync
+PACKAGE_VERSION=${1:-036812b2}
+GO_VERSION=1.17.1
+PACKAGE_URL=https://github.com/golang/sync.git
+
+dnf install git wget -y
+
+mkdir -p /home/tester/output
+cd /home/tester
+
+wget https://golang.org/dl/go$GO_VERSION.linux-ppc64le.tar.gz
+rm -rf /home/tester/go && tar -C /home/tester -xzf go$GO_VERSION.linux-ppc64le.tar.gz
+rm -f go$GO_VERSION.linux-ppc64le.tar.gz
+export GOPATH=/home/tester/go
+export PATH=$PATH:$GOPATH/bin
+
+mkdir -p $GOPATH/src/github.com/golang
+cd $GOPATH/src/github.com/golang
+rm -rf $PACKAGE_NAME
+
+OS_NAME=$(cat /etc/os-release | grep ^PRETTY_NAME | cut -d= -f2)
+
+
+if ! git clone $PACKAGE_URL $PACKAGE_NAME; then
+        echo "------------------$PACKAGE_NAME:clone_fails---------------------------------------"
+        echo "$PACKAGE_URL $PACKAGE_NAME" > /home/tester/output/clone_fails
+    echo "$PACKAGE_NAME  |  $PACKAGE_URL |  $PACKAGE_VERSION | $OS_NAME | GitHub | Fail |  Clone_Fails" > /home/tester/output/version_tracker
+        exit 1
+fi
+
+cd $PACKAGE_NAME
+git checkout $PACKAGE_VERSION
+
+INSTALL_SUCCESS="false"
```

Comment:
Any specific reason for using this INSTALL_SUCCESS variable ?
We are assigning it but not using it .

Reply:
Okay. Not required, Removing.

Reply:
Default set to false.

Reply:
I am not concern about setting default values.
My concern is, Where exactly that variable is being used ?

Its always being set, but not at all used. Not in conditions and not as return value.

Reply:
Okay. Removed.